src/shared/database_DEPRECATED/database.providers.ts

```diff
@@ -0,0 +1,20 @@
+import { createConnection } from 'typeorm';
+import { config } from '@src/config';
+
+export const databaseProviders = [
+  {
+    provide: config.dataBase.provider,
+    useFactory: async () => await createConnection({
+      type: 'mysql',
+      host: config.dataBase.host,
+      port: config.dataBase.port,
+      username: config.dataBase.user,
+      password: config.dataBase.password,
+      database: config.dataBase.name,
+      entities: [
+        __dirname + '/../../app/**/*.entity{.ts,.js}',
```

Comment:
it is better to use path.join here
import path from 'path';
...
path.join(__dirname, '../../app/**/*.entity{.ts,.js}')
...
at least, just try if it works for you